environment-dev.yml

```diff
@@ -54,3 +53,4 @@ dependencies:
   - xorg-libxrender
   - pip:
       - mknotebooks
+      - mkdocstrings[python]
```

Comment:
If possible, could we move `mkdocstrings` to the conda-installed section? I am still unsure if `pip` still respects `conda`-installed dependencies.

Reply:
I tried but failed https://github.com/pyjanitor-devs/pyjanitor/commit/8cf8609e37e960352d4cdfcaca9b8b20ccca19e8

Conda seems can't work well with extra installing like 'package[extra]'

Maybe we could install directly mksdocstrings and its extra package (mkdocstrings-python) via conda.

https://github.com/pyjanitor-devs/pyjanitor/commit/8cf8609e37e960352d4cdfcaca9b8b20ccca19e8

Reply:
fix via installing `mkdocstrings-python`